_posts/2019-06-07-python-datascience-with-openfaas.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Serving Data Science Models with OpenFaaS
```

Comment:
Change to suggestion:

Build highly scaleable, fault tolerant neural networks with OpenFaaS and PyTorch

Based upon the content of the post.

> The name classifier function uses the neural network implementation in PyTorch

You can update the description to match this and look how we have made us of the author's name in the description on other recent posts.

Reply:
The article does deploy a pytorch neural net, but the related template is not restricted to just that case.  I was hoping that the title could be a little more broad so that it would be more obvious that this pattern can be applied in general